src/System.Runtime.Numerics/src/System/Numerics/BigCalc.SquMul.cs

```diff
@@ -0,0 +1,356 @@
+﻿using System.Diagnostics;
+using System.Security;
+
+namespace System.Numerics
+{
+    internal static partial class BigCalc
```

Comment:
I know it's an internal type, but I think it would be better for code browsing if it was called BigIntegerCalculator, BigIntegerHelpers, BigIntegerMath, or something like that.